src/status_im/utils/datetime.cljs

```diff
@@ -29,28 +31,54 @@
 
 (def time-zone-offset (hours (- (/ (.getTimezoneOffset ^js (js/Date.)) 60))))
 
-;; detects if given locale symbols timeformat generates AM/PM ("a")
-(defn- is24Hour-locsym [^js locsym]
+;;;; Utilities
+
+(defn- is24Hour-locsym
+  "Detects if given locale symbols timeformat generates AM/PM ('a')."
+  [^js locsym]
   (not (s/includes?
         (nth (.-TIMEFORMATS locsym) 2)
         "a")))
 
-;; returns is24Hour from device or from given locale symbols
-;; whenever we get non-nil value use it, else calculate it from the given locale symbol
-(defn- is24Hour [locsym]
+(defn- is24Hour
+  "Returns is24Hour from device or from given locale symbols. Whenever we get
+  non-nil value use it, else calculate it from the given locale symbol."
+  [^js locsym]
   (if-some [fromdev (status/is24Hour)]
     fromdev
     (is24Hour-locsym locsym)))
 
-;; time formats
-(defn- short-time-format [locsym] (if (is24Hour locsym) "HH:mm" "h:mm a"))
-(defn- time-format [locsym] (if (is24Hour locsym) "HH:mm:ss" "h:mm:ss a"))
+;;;; Time formats
+
+(defn- short-time-format
+  [^js locsym]
+  (if (is24Hour locsym)
+    "HH:mm"
+    "h:mm a"))
+
+(defn- time-format
+  [^js locsym]
+  (if (is24Hour locsym)
+    "HH:mm:ss"
+    "h:mm:ss a"))
+
+;;;; Date formats
 
-;; date formats
 (defn- short-date-format [_] "dd MMM")
-(defn- medium-date-format [^js locsym] (nth (.-DATEFORMATS locsym) 2)) ; get medium format from current locale symbols
 
-;; date-time formats
+(defn- datetime-within-one-week-format
+  [^js locsym]
+  (if (is24Hour locsym)
+    "E HH:mm"
+    "E h:mm a"))
+
+(defn- medium-date-format
+  "Get medium format from current locale symbols."
```

Comment:
what is medium format? 🤨

Reply:
@J-Son89, if you look at the diff, you'll see that I tried only to reformat and move the original comment to the docstring. But ideally we could do a better job at describing some of these things.